Userland/Libraries/LibWasm/AbstractMachine/Validator.h

```diff
@@ -33,7 +33,7 @@ struct Context {
 
 struct ValidationError : public Error {
     ValidationError(DeprecatedString error)
-        : Error(Error::from_string_view(error))
+        : Error(Error::from_string_view(error.view()))
```

Comment:
Holllld up. Shouldn't this use the view into the error_string member? It might need updated after the fact...

Reply:
Doesn't really matter, that String move is just a copy without refcount changes :shrug: